config/ember-try.js

```diff
@@ -1,55 +1,49 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = async function() {
+module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-3.12',
+        name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
-            'ember-source': '~3.12.0',
+            'ember-source': '~3.24.3',
           },
         },
       },
       {
-        name: 'ember-lts-3.16',
+        name: 'ember-lts-3.28',
```

Comment:
Can you also add `ember-lts-4.4` and `ember-lts-4.8` (both here and in ci.yml) now that they are both LTS?

Reply:
fixed in 9b060a19a1e4698862945f731c2a3f257233ab9a